ui/main/src/app/business/builtInTemplates/task/usercard/taskUserCardTemplate.ts

```
@@ -552,9 +552,12 @@ export class TaskUserCardTemplate extends BaseUserCardTemplate {
         const durationInMinutes: number = parseInt(
             (<HTMLInputElement>document.getElementById('durationInMinutes')).value
         );
-        const minutesForReminder: number = parseInt(
+        let minutesForReminder: number = parseInt(
             (<HTMLInputElement>document.getElementById('minutesForReminder')).value
         );
+        if (minutesForReminder == null || isNaN(minutesForReminder)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)